internal/statestore/backfill.go

```diff
@@ -28,7 +29,8 @@ import (
 )
 
 const (
-	allBackfills = "allBackfills"
+	backfillLastAckTime = "backfill_last_ack_time"
```

Comment:
This also needs to be set during backfill creation, so that a server which never starts up can be cleaned up.

Reply:
Yes, that's true, made a TODO for this I wanted to make this as part of separate PR.
But I think can be added here also.

Reply:
Added additional call acknowledgeBackfill to StateStore CreateBackfill.